langlib/lang.array/src/main/java/org/ballerinalang/langlib/array/Remove.java

```diff
@@ -18,10 +18,8 @@
 
 package org.ballerinalang.langlib.array;
 
-import io.ballerina.runtime.api.utils.TypeUtils;
```

Comment:
We can remove unnecessary new lines in the class.

Reply:
I'll remove them